backend/ts/src/main/scala/aqua/backend/ts/TypeScriptCommon.scala

```diff
@@ -0,0 +1,104 @@
+package aqua.backend.ts
+
+import aqua.backend.air.FuncAirGen
+import aqua.model.transform.res.FuncRes
+import aqua.types.*
+import cats.syntax.show.*
+
+object TypeScriptCommon {
+
+  def typeToTs(t: Type): String = t match {
+    case OptionType(t) => typeToTs(t) + " | null"
+    case ArrayType(t) => typeToTs(t) + "[]"
+    case StreamType(t) => typeToTs(t) + "[]"
+    case pt: ProductType =>
+      "[" + pt.toList.map(typeToTs).mkString(", ") + "]"
+    case st: StructType => 
+      s"{${st.fields.map(typeToTs).toNel.map(kv => kv._1 + ":" + kv._2).toList.mkString(";")}}"
+    case st: ScalarType if ScalarType.number(st) => "number"
+    case ScalarType.bool => "boolean"
+    case ScalarType.string => "string"
+    case lt: LiteralType if lt.oneOf.exists(ScalarType.number) => "number"
+    case lt: LiteralType if lt.oneOf(ScalarType.bool) => "boolean"
+    case lt: LiteralType if lt.oneOf(ScalarType.string) => "string"
+    case _: DataType => "any"
+    case at: ArrowType => fnDef(at)
+  }
+
+  def fixupArgName(arg: String): String =
```

Comment:
what if we already have `peer_` or `config_` variable?

Reply:
I've added a TODO: